NewSpring/Rock
# Fixed bug where templates for content channel items with multiple parent items were pulling data from the wrong parent item

## DESCRIPTION

### What does this PR do, or why is it needed?

When we wrote the original lava library for the new newspring.cc, we did so under the premise that content channel items would have, at most, a single parent item - since that has been the case up until now. 

As we have begun using content channel items for aspects of the app, such as personalization campaigns, we've realized a need items to have multiple parents. This introduced a problem where our lava was not specific enough to explicitly define _which_ channel to get parent item data from for things like sermon series, devotionals, and others.

The way we have fixed this issue is by creating a Content Channel Type Channel Attribute called `PrimaryParentChannel` - which gives us the ability to, on a per channel basis, define which channel we should be getting parent data from for any item in that channel.

For example, the attribute on the `Sermons` channel is set for `NewSpring - Sermon Series` is the primary parent - so in our lava templates, when we get parent data like series title, colors, imagery, we know that if there is more than one parent item, we should get the data from the parent that is in the `NewSpring - Sermon Series` channel.

### How do I test this PR?

1. Add any sermon or devotional as a child of any item in the `App - Campaigns` channel.
2. View the list/entry pages of the series/sermon or study/devotional and verify that all parent titles, imagery, brand colors, and links are correct.

## TODO

- [] Set a relevant reviewer

## REVIEW


> The purpose of PR Review is to _improve the quality of the software._
